providers/gpgpu/units/jobs.pxu

```diff
@@ -4,31 +4,31 @@ plugin: shell
 estimated_duration: 300
 requires:
     graphics_card.vendor == 'NVIDIA Corporation'
-    package.name == 'cuda-toolkit'
+    snap.name == 'gpu-burn'
     uname.machine == 'x86_64'
 _summary: NVIDIA GPGPU stress testing
-command: gpu_burn.sh
+command: gpu-burn 14400 |  grep -v -e '^[[:space:]]*$' -e "errors:" -e "Summary at"
```

Comment:
I see that this partially replaces the old script but please avoid hiding the output of commands (I know, this is not new code, but lets fix it). Please `tee` the output so that it can be read from the session and `set -eo pipefail` for now